CMakeLists.txt

```diff
@@ -233,8 +233,6 @@ IF(SPARK_CONNECTOR)
   ENDIF(PYTHON)
 ENDIF(SPARK_CONNECTOR)  
 
-OPTION(SIGNCODE "Sign the MSI installer" OFF)
```

Comment:
Is this intended to be here as the commit is for centos, and in any case shouldn't we be signing the msi install?

Reply:
confirmed this is handled later and this is just removing dead code.